spec/forms/openid_connect_authorize_form_spec.rb

```diff
@@ -77,9 +99,20 @@
       let(:response_type) { nil }
 
       it 'is unsuccessful and has error messages' do
-        expect(result[:success]).to eq(false)
-        expect(result[:client_id]).to eq(client_id)
-        expect(result[:errors]).to be_present
+        form_response = instance_double(FormResponse)
+
+        extra_attributes = {
+          client_id: client_id,
+          redirect_uri: "#{redirect_uri}?error=invalid_request&error_description=" \
+                        "Response+type+is+not+included+in+the+list&state=#{state}",
+        }
+
+        errors = { response_type: ['is not included in the list'] }
+
+        expect(FormResponse).to receive(:new).
+          with(success: false, errors: errors, extra: extra_attributes).and_return(form_response)
+        expect(result).to eq form_response
+        expect(form.client_id).to eq(client_id)
```

Comment:
isn't this line handled by making sure `client_id` is in the `extra_attributes` hash?